File: 0073-set-matrix-zeroes/0073-set-matrix-zeroes.py
class Solution:
    def setZeroes(self, m: List[List[int]]) -> None:
        """
        Do not return anything, modify matrix in-place instead.
        """
        l=[]
        for i in range(len(m)):
            a=0
            for j in range(len(m[i])):
                if(a==0):
                    for k in range(len(m[i])):
                        if(m[i][k]==0):
                            a=1
                            break
                    if(a==0):
                        break
                if(a==1):
                    if(m[i][j]==0):
                        l.append(j)
                    m[i][j]=0
        for i in range(len(m)):
            for j in range(len(l)):
                m[i][l[j]]=0
                

File: 0073-set-matrix-zeroes/0073-set-matrix-zeroes.cpp
class Solution {
public:
    void setZeroes(vector<vector<int>>& m) 
    {
        int i,j,k;
        vector<int>v;
        for(i=0;i<m.size();i++)
        {
            int a=0;
            for(j=0;j<m[i].size();j++)
            {
                if(a==0)
                {
                    for(k=0;k<m[i].size();k++)
                    {
                        if(m[i][k]==0)
                        {
                            a=1;
                            break;
                        }
                    } 
                    if(a==0)
                    {
                        break;
                    }
                }
                if(a==1)
                {
                    if(m[i][j]==0)
                    {
                        v.push_back(j); 
                    }
                    m[i][j]=0;
                }
                
            }
        }
        for(i=0;i<m.size();i++)
        {
            for(j=0;j<v.size();j++)
            {
                m[i][v[j]]=0;
            }
        }
        
    }
};